en/1-who-we-are.md

```diff
@@ -1,12 +1,13 @@
 
 # 1. About Us
 
-Meeds is a collaborative effort led by diverse Open Source Software (OSS) communities, dedicated to creating a decentralized digital world that resists centralized corporate control and fosters a free and open environment.
+Meeds is a collaborative effort led by diverse open-source software (OSS) communities dedicated to creating a decentralized digital world that resists centralized corporate control and promotes a free and open environment.
```

Comment:
Foster is more about taking care of something or someone. 
Promote is making others aware of a thing. So if you promote an item or an idea, you are letting other people know about it. Promoting is often associated with advertisements. So, "promote" is a preferable term to use.

Reply:
Thanks @Amby01 but green color is for changes added, and red is for changes removed :-)

Here the change is exactly what you described. I'm using the promote verb.